openedx/core/djangoapps/programs/utils.py

```diff
@@ -673,12 +673,19 @@ def get_certificates(user, extended_program):
         certificates.append({
             'type': 'program',
             'title': extended_program['title'],
-            'url': program_credentials[0]['certificate_url'],
+            'url': get_program_certificate_url(program_credentials[0]['certificate_url']),
         })
 
     return certificates
 
 
+def get_program_certificate_url(certificate_url):
```

Comment:
@ahsan-ul-haq Can we rename this to `get_logged_in_program_certificate_url` to be more clear about what this method is doing?